docs/content/dagster-plus/managing-deployments/branch-deployments/change-tracking.mdx

```diff
@@ -17,6 +17,8 @@ Branch Deployments compare asset definitions in the branch deployment against th
 
 You can also apply filters to show only new and changed assets in the UI. This makes it easy to understand which assets will be impacted by the changes in the pull request associated with the branch deployment.
 
+**Note:** the default main deployment is `prod`. To configure a different deployment as the main deployment, [create a branch deployment using the dagster-cloud CLI](/dagster-plus/managing-deployments/branch-deployments/using-branch-deployments) and specify it using the optional `--base-deployment-name` parameter.
```

Comment:
just an FYI if this comes up in support- this is true in most, but not all cases. I'm not sure it's worth getting into all of the cases here though